### PR TITLE
Add voxel_min_points param for voxel filter (ROS2)

### PR DESCRIPTION
**Pull request by akiroz**
See #160

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ rgbd_obstacle_layer:
     inf_is_valid: false          #default false, for laser scans
     clear_after_reading: true    #default false, clear the buffer after the layer gets readings from it
     voxel_filter: true           #default off, apply voxel filter to sensor, recommend on 
+    voxel_min_points: 10         #default 0, minimum points per voxel for voxel filter
```
Comment (SteveMacenski) [inline README.md]: Please change this to `0`.

File: src/measurement_buffer.cpp [hunk 2]
```diff
@@ -154,6 +154,7 @@ void MeasurementBuffer::BufferROSCloud(
       sor.setDownsampleAllData(false);
       float v_s = static_cast<float>(_voxel_size);
       sor.setLeafSize(v_s, v_s, v_s);
+      sor.setMinimumPointsNumberPerVoxel((unsigned int) _voxel_min_points);
```
Comment (SteveMacenski) [inline src/measurement_buffer.cpp]: please use `static_cast<unsigned int>(_voxel_min_points);`

**Review by SteveMacenski — CHANGES_REQUESTED** [review]
Same items from ROS1

**Review by SteveMacenski — APPROVED** [review]

**Comment by nickvaras** [comment]
There's an issue with minimum points per voxel in PCL, so just wanted to confirm what I'd expect, which is that the feature added in this PR doesn't really have any effect when applied?? Unless it's been fixed in newer versions of PCL...

**Comment by SteveMacenski** [comment]
I'm not sure - I haven't tested this personally, @akiroz I would have thought did since they proposed it

**Comment by akiroz** [comment]
We actually ended up not using this param in the end but I believe it's been fixed in PCL upstream and planned for the coming 1.12 release.

**Comment by SteveMacenski** [comment]
This appears to have potentially broken things per https://github.com/SteveMacenski/spatio_temporal_voxel_layer/issues/187 and https://github.com/SteveMacenski/spatio_temporal_voxel_layer/issues/199

**Pull request merged**